impl_test.go

```diff
@@ -34,7 +34,7 @@ func TestFindInterface(t *testing.T) {
 	}
 
 	for _, tt := range cases {
-		path, id, err := findInterface(tt.iface)
+		path, id, err := findInterface(tt.iface, ".")
```

Comment:
Those are the values that were hardcoded before this change.  See [here](https://github.com/josharian/impl/blob/43306ba79c4e3d1a038db0365b189ad4cca413c6/impl.go#L66) and [here](https://github.com/josharian/impl/blob/43306ba79c4e3d1a038db0365b189ad4cca413c6/impl.go#L101)

The other requested changes have been addressed in my latest commits.